R/check-superseded-deps.R

```diff
@@ -0,0 +1,71 @@
+
+#' Check whether the package depends on discouraged dependencies.
+#'
+#' @inheritParams pkg_has_no_scrap
+#' @return Names of any packages which should not be depended upon; otherwise an empty
+#' character.
+#' @noRd
+pkgchk_has_superseded_deps <- function (s) {
+
```

Comment:
It does not work yet cf https://github.com/ropensci-review-tools/pkgstats/issues/21 (I could work around the current output but am waiting for the decision!)